CodePush.js

```diff
@@ -100,12 +100,16 @@ const getConfiguration = (() => {
 })();
 
 async function getCurrentPackage() {
-  const localPackage = await NativeCodePush.getCurrentPackage();
-  if (localPackage) {
-      localPackage.failedInstall = await NativeCodePush.isFailedUpdate(localPackage.packageHash);
-      localPackage.isFirstRun = await NativeCodePush.isFirstRun(localPackage.packageHash);
+  return await getUpdateMetadata(CodePush.UpdateState.LATEST);
+}
+
+async function getUpdateMetadata(updateState) {
+  const updateMetadata = await NativeCodePush.getUpdateMetadata(updateState || CodePush.UpdateState.RUNNING);
+  if (updateMetadata) {
+      updateMetadata.failedInstall = await NativeCodePush.isFailedUpdate(updateMetadata.packageHash);
+      updateMetadata.isFirstRun = await NativeCodePush.isFirstRun(updateMetadata.packageHash);
```

Comment:
I think we need to agree on whether we are using 2 spaces or 4 spaces for tabs at some point :) The above are using 4 spaces, but everywhere else is 2 spaces.